### ISSUE TITLE
Put Height Warning In 'Agent Settings' page

### ISSUE THREAD

**Issue by sinistersnare** [Unity-Technologies/NavMeshComponents#84]
Hi, If I make an agent with a `height` of `n`, and `step height < n`, a warning will appear in the inspector's Nav Mesh Surface section. I feel like it would be a nice enhancement to also have this warning in the Navigation pane itself. Just thought it would be a nice-to-have. If this is out of scope, then feel free to close this. Thanks!

**Comment by turadr** [comment]
Indeed, the Navigation > Agents window could use that warning message as well. Thank you for the suggestion.
